lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -905,7 +907,8 @@ def test_no_problems(self, _get_current_task):
         ])
 
     @patch('lms.djangoapps.instructor_task.tasks_helper.runner._get_current_task')
-    def test_single_problem(self, _get_current_task):
+    def test_problem_grade_report_all_learners(self, _get_current_task):
```

Comment:
@DawoudSheraz I can explain why this test was updated and why I added another submission. 

This test was testing that grade report is created with the submission of all the learners having a single problem. In order to fit that test in my current scenario, I also created a verified enrollment and attempted the problem. In the end, I have verified that the verified learner is also included in the report and is not skipped. 

Given that, I changed the name of the test as well so that its easier to understand. I can add a docstring as well.